### ISSUE TITLE
release 2.0 in supermarket

### ISSUE THREAD

**Issue by chris-rock** [dev-sec/chef-os-hardening#148]
@artem-sidorenko Do you see anything that prevents us to release the current master as v 2.0.0? Supermarket still uses 1.4.1

**Comment by artem-sidorenko** [comment]
My view/plan for 2.0 was:
 1. fix the tests/get CI green - currently only the audit stuff is missing, so this point might be seen optional
 2. fix the attributes, which depend on other attributes (see [this](https://github.com/dev-sec/chef-ssh-hardening/issues/140#issuecomment-267779720) for details). Because of changing cookbook behavior this is a must in my eyes for 2.0 

I try to cover the second thing (and ideally the first too) this/next week, ideally prior to the chef 13 release

**Comment by chris-rock** [comment]
@artem-sidorenko just to make sure, I am talking about os-hardening 2.0

**Comment by artem-sidorenko** [comment]
@chris-rock right, this comment just describes the problem in detail. We had this problem in ssh-hardening in one place as far I remember, in os-hardening there are several places with this problem

**Comment by artem-sidorenko** [comment]
@chris-rock Example, if I want to properly enable IPv6 I have to do something like this:

```ruby
# Enable IPv6
override['network']['ipv6']['enable'] = true
override['sysctl']['params']['net']['ipv6']['conf']['all']['disable_ipv6'] = 0

override['sysctl']['params']['net']['ipv6']['conf']['default']['accept_ra_rtr_pref'] = 1
override['sysctl']['params']['net']['ipv6']['conf']['default']['accept_ra_pinfo'] = 1
override['sysctl']['params']['net']['ipv6']['conf']['default']['accept_ra_defrtr'] = 1
override['sysctl']['params']['net']['ipv6']['conf']['default']['autoconf'] = 1
override['sysctl']['params']['net']['ipv6']['conf']['default']['dad_transmits'] = 1
override['sysctl']['params']['net']['ipv6']['conf']['default']['max_addresses'] = 16

override['sysctl']['params']['net']['ipv6']['conf']['all']['accept_ra'] = 1
override['sysctl']['params']['net']['ipv6']['conf']['default']['accept_ra'] = 1
```

**Comment by chris-rock** [comment]
Makes sense.

**Comment by artem-sidorenko** [comment]
As soon as #149 and #150 are reviewed and merged we can release the 2.0.0 from my point of view

**Comment by artem-sidorenko** [comment]
@atomic111 @chris-rock once #151 is merged I would like to release 2.0.0. What do you think?

**Comment by artem-sidorenko** [comment]
2.0.0 is released
